src/main/cpp/RLBotInterface/src/RLBotMessages/MessageStructs/Message.hpp

```diff
@@ -26,7 +26,7 @@ struct MessageBase
 protected:
 	MessageBase(MessageType type, bool hasCallback) : Type(type), HasCallback(hasCallback)
 	{
-		static std::atomic_uint currentID = 0;
+		static std::atomic_uint currentID = {0};
```

Comment:
Is there a reason to not just invoke the constructor directly? I checked with godbolt and the generated code is the same but its a lot easier to read.

```cpp
static std::atomic_uint currentID(0);
```

Reply:
Sorry about the huge delay. Yeah, that would be a lot better - I don't write much C++ so I didn't really think about the cleanest way to write it at the time. Thanks for the suggestion!